paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -441,7 +452,12 @@ internal class CustomerSheetViewModel @Inject constructor(
     private fun transitionToAddPaymentMethod(isFirstPaymentMethod: Boolean) {
         val paymentMethodCode = PaymentMethod.Type.Card.code
 
-        val formArguments = createFormArguments(card)
+        val formArguments = FormArgumentsFactory.create(
+            paymentMethod = card,
+            configuration = configuration,
+            merchantName = configuration.merchantDisplayName
```

Comment:
Can we inject a `PackageNameProvider` to avoid `Application` usage?

Reply:
That is a slightly bigger change, I can do this in a future PR

Reply:
I added a comment to make sure I come back to it.